app/models/application.rb

```diff
@@ -62,7 +62,7 @@ class Application < ApplicationRecord
     active: "active",
     deferred: "deferred",
     withdrawn: "withdrawn",
-  }
+  }, _prefix: "training_status"
```

Comment:
thinking about this, I think keeping it without a prefix is preferable as the actions are quite clear, but happy to chat through this